app/src/main/java/pl/futurecollars/invoicing/db/DatabaseConfiguration.java

```diff
@@ -0,0 +1,49 @@
+package pl.futurecollars.invoicing.db;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import pl.futurecollars.invoicing.db.memory.FileBasedDatabase;
+import pl.futurecollars.invoicing.db.memory.InMemoryDatabase;
+import pl.futurecollars.invoicing.service.FileService;
+import pl.futurecollars.invoicing.service.IdService;
+import pl.futurecollars.invoicing.service.JsonService;
+
+@Slf4j
+@Configuration
+public class DatabaseConfiguration {
```

Comment:
w momencie, kiedy przeszliśmy na @Configuration to klasa `PathConfiguration` już nie jest używana, można usunąć